math/dlib/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake  1.1
+PortGroup           cxx11  1.1
+
+github.setup        davisking dlib v19.17
+name                dlib
```

Comment:
The `name` line is redundant because it is added by the `github.setup` command and should be removed.